recipe/meta.yaml

```diff
@@ -100,8 +100,9 @@ outputs:
         - zlib {{ zlib }}
         - zstd {{ zstd }}
         # Use pins to control cos6/cos7 match
-        - libgcc-ng  {{ libgcc }}
-        - libstdcxx-ng {{ libstdcxx }}
+        - libgcc-ng  {{ libgcc }}          # [x86_64 and c_compiler_version == "7.2.*"]
+        - libstdcxx-ng {{ libstdcxx }}     # [x86_64 and c_compiler_version == "7.2.*"]
+        - openssl 1.1.1k                   # [x86_64 and c_compiler_version == "7.2.*"]
```

Comment:
Is this needed here? I think the `grpc-cpp` change will pull in the openssl pinning.

Reply:
Nevermind, grpc-cpp is only pinning in the host section.